api/config/index.js

```diff
@@ -1,12 +1,12 @@
-// default this
-module.exports = {
-  mysql: {
-    host: process.env.DB_HOST || 'localhost',
-    port: process.env.DB_PORT || 3306,
-    user: process.env.DB_USER || 'root',
-    password: process.env.DB_PASSWORD || 'root',
-    database: process.env.NODE_ENV === 'development' ? 'SikhiFM' : 'sikhiFMDB',
-    dateStrings: true,
-    connectionLimit: process.env.DB_POOL_SIZE,
-  },
+// config for SikhiFM Database
+const metadata = {
+  host: 'localhost',
+  user: process.env.DB_USER || 'root',
+  password: process.env.DB_PASSWORD || 'root',
+  database: process.env.NODE_ENV === 'development' ? 'sikhifm_db' : 'sikhifm_db',
+  post: 3306,
```

Comment:
should be port

Reply:
should make it so if someone else is running their DB on different port, they can specify. something like 
```
port: process.env.DB_PORT || 3306',
```